sources/api/corndog/README.md

```diff
@@ -9,4 +9,4 @@ It sets kernel-related settings, for example:
 
 ## Colophon
 
-This text was generated from `README.tpl` using [cargo-readme](https://crates.io/crates/cargo-readme), and includes the rustdoc from `src/main.rs`.
\ No newline at end of file
+This text was generated from `README.tpl` using [cargo-readme](https://crates.io/crates/cargo-readme), and includes the rustdoc from `src/main.rs`.
```

Comment:
This seems to be a problem in cargo-readme where running `cargo readme > README.md` can produce a different result than when using `cargo_readme::generate_readme()` 😞 

There's a call to `template.trim_end_matches("\n")` here: https://github.com/livioribeiro/cargo-readme/blob/11ce95834c5ef52922174a22cd91e9d1868cf3d4/src/readme/template.rs#L54

I suspect that when `cargo readme` is called a newline is added at the end at some other part of the code.

Anyway, I made a change to `generate_readme` to append a newline. Hopefully that is good enough.

It might make sense to try to fix this in cargo-readme, but it looks like there's not much development there anymore. Let me know if you want me to revert these newlines.